crates/rpc/rpc-engine-api/tests/it/payload.rs

```diff
@@ -108,42 +108,6 @@ fn payload_validation() {
     payload_with_invalid_txs.transactions.iter_mut().for_each(|tx| {
         *tx = Bytes::new();
     });
-    let payload_with_invalid_txs =
-        try_payload_v1_to_block::<TransactionSigned>(payload_with_invalid_txs);
+    let payload_with_invalid_txs = payload_with_invalid_txs.try_into_block::<TransactionSigned>();
     assert_matches!(payload_with_invalid_txs, Err(PayloadError::Decode(RlpError::InputTooShort)));
-
-    // Non empty ommers
-    let block_with_ommers = transform_block(block.clone(), |mut b| {
-        b.body.ommers.push(random_header(&mut rng, 100, None).unseal());
-        b
-    });
-    assert_matches!(
```

Comment:
I removed these, because this weren't useful because 

we always enforce this

https://github.com/paradigmxyz/reth/blob/eebdd0d488f100f7d4e55642a1f40457aef0826e/crates/payload/validator/src/lib.rs#L125-L126